src/components/utility/Typography/Typography.js

```diff
@@ -1,29 +1,35 @@
-import React from 'react';
+import React from "react";
+import styles from "./Typography.module.css";
 
 export const TypographyElements = {
-  "body1": "p",
-  "body2": "p",
-  "span": "span",
-  "h1Small": "h1",
-  "h1Large": "h1",
-  "h2": "h2",
-  "h3": "h3",
-}
+  body1: "p",
+  body2: "p",
+  span: "span",
+  h1: "h1",
+  h2: "h2",
+  h3: "h3",
+};
 
-export default function Typography(
-  { elType, className = "", id = "", children }
-) {
+export default function Typography({
+  elType,
+  className = "",
+  id = "",
+  children,
+}) {
   if (!Object.keys(TypographyElements).includes(elType)) {
-    throw TypeError("Unsupported element type for Typography component")
-  };
+    throw TypeError("Unsupported element type for Typography component");
+  }
 
-  className = `${elType} ${className}`;
+  var typographyStyle = `${styles[elType]}`;
+  if (className) {
+    typographyStyle += `${styles[className]}`;
```

Comment:
I like the reformatting of how to concat the typography style and the class name, BUT the `className` prop is probably NOT something inside typography styles (rather in one of the components _using_ Typography, to do things like align it) and should not be matched against the `styles` object.

Reply:
We can have styles like this as well, https://github.com/poulami-saha/Global-Code-Queens-MyTeam-Website/pull/7